src/test/java/org/helpberkeley/memberdata/ControlBlockTestBase.java

```diff
@@ -157,7 +157,7 @@ public void tooManyOpsManagerValueSeparatorTest() {
 
         Throwable thrown = catchThrowable(workflowParser::drivers);
         assertThat(thrown).isInstanceOf(MemberDataException.class);
-        assertThat(thrown).hasMessageContaining("does not match \"username | phone\"");
+        assertThat(thrown).hasMessageContaining("does not match \"username\" or \"username | phone\"");
```

Comment:
Convert this error string to use MessageFormat and a constant from ControlBlock.   This is one of the ways of avoiding string duplication between production and test code.